dev/create-release/release-build.sh

```diff
@@ -330,6 +336,14 @@ if [[ "$1" == "package" ]]; then
         --output $PYTHON_CONNECT_DIST_NAME.asc \
         --detach-sig $PYTHON_CONNECT_DIST_NAME
       shasum -a 512 $PYTHON_CONNECT_DIST_NAME > $PYTHON_CONNECT_DIST_NAME.sha512
+
+      PYTHON_CLIENT_DIST_NAME=pyspark-client-$PYSPARK_VERSION.tar.gz
+      cp spark-$SPARK_VERSION-bin-$NAME/python/dist/$PYTHON_CLIENT_DIST_NAME .
```

Comment:
Unfortunately this doesn't work...

Reply:
This is just copy so we must have generated these packages somewhere, and maybe that place does not generate the pyspark-client?